lapce-data/src/data.rs

```diff
@@ -133,7 +137,71 @@ impl LapceData {
         let latest_release = Arc::new(None);
 
         let dirs: Vec<&PathBuf> = paths.iter().filter(|p| p.is_dir()).collect();
-        let files: Vec<&PathBuf> = paths.iter().filter(|p| p.is_file()).collect();
+
+        // Storage for parsed file, line and column.
+        let mut files: Vec<PathBuf> = vec![];
+        let mut lines: Vec<Line> = vec![];
+        let mut columns: Vec<LineCol> = vec![];
+
+        // extract file, line and column from the path
+        paths.iter().for_each(|a| {
+            let argument = a
+                .file_name()
```

Comment:
It seems this function cuts everything leaving only the file name (https://doc.rust-lang.org/std/path/struct.Path.html#method.file_name)